src/app/board/page.js

```diff
@@ -0,0 +1,23 @@
+"use client";
+import Title from "@/components/Title";
+import Board from "@/components/board/Board";
+
+const Meet = () => {
+  const boardMembers = Array.from({ length: 12 }, () => ({
+    name: "Evan Littlecat",
+    position: "Position Name",
+  }));
+
+  return (
+    <div className="flex flex-col items-center justify-start min-h-screen pt-10">
+      <Title text="Meet the Board" />
+      <div className="grid grid-cols-4 gap-12 mt-10">
```

Comment:
y-axis gap looks fine, but we need a little more space horizontally

Reply:
![image](https://github.com/acm-ucr/isa-website/assets/93837219/7f2c20d3-f08b-4ce5-8a10-4ee37731715f)
Is this better? I added some more horizontal space and increased the top padding slightly

Reply:
add a little more spacing

Reply:
yeah horizontally

Reply:
How about this?
![image](https://github.com/acm-ucr/isa-website/assets/93837219/73827e6f-c5cd-4326-8bb7-18ea652cc726)

Reply:
that should be good 👍👍